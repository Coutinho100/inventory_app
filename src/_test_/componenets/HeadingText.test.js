import React from 'react';
import HeadingText from '../../components/Component/HeadingText';
import { View, Text } from 'react-native';
import { configure, mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


describe("<Text render/>", () => {

    it('renders the length of view', () => {
        const wrapper = shallow(<HeadingText />);
        expect(wrapper.find(Text).length).toEqual(1);
    })

});
