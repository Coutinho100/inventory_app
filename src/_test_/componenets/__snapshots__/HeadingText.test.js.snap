// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text render/> renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "fontSize": 20,
        "fontWeight": "bold",
      },
      undefined,
    ]
  }
/>
`;
